package.json

```diff
@@ -36,20 +63,21 @@
         "ms-kubernetes-tools.vscode-kubernetes-tools"
     ],
     "devDependencies": {
-        "typescript": "^2.6.1",
-        "vscode": "^1.1.6",
-        "tslint": "^5.8.0",
+        "@types/js-yaml": "^3.12.1",
+        "@types/mocha": "^2.2.42",
         "@types/node": "^7.0.43",
-        "@types/mocha": "^2.2.42"
+        "tslint": "^5.8.0",
+        "typescript": "^2.6.1",
```

Comment:
Suggest we upgrade to pin a more recent version of TypeScript - with the new dependency, I'm getting errors `Cannot find name 'unknown'` in `vscode-azureextensionui` - we should switch to TypeScript 3.x or above.  (Which I should have done anyway in the first place!)

Reply:
Updated.